_posts/0000-01-02-vampyteuth.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![image](https://user-images.githubusercontent.com/94151404/161149645-1609af6e-0c12-44bd-8117-7bdac895f954.png)

Reply:
I want to code my pants off---LEZZZZZZ GOOOOOO